packages/wallet-core/src/types.ts

```diff
@@ -107,6 +107,23 @@ export type CloseChannel = _Objective<
   {
     targetChannelId: string;
     fundingStrategy: FundingStrategy;
+    /**
+     * Collaboratively concluding a channel on-chain involves the following:
+     * 1. Submit a conclusion proof to the NitroAdjudicator.
+     * 2. Push the outcome to all AssetHolders.
+     * 3. Transfer out of the channel for all asset holders.
```

Comment:
Do you expect `tsSubmitterOrder[0]` will be responsible for all of step (3)? Or do you imagine an iterative approach, ie.
```
for (const idx of tsSubmitterOrder) {
  // participant[idx] withdraws their assets until they have no funds in the channel
}
```

Reply:
In the medium term, it likely makes sense for the primary transaction submitter to only withdraw their own assets. In the short term, the primary transaction submitter is responsible for all 3 steps.

Reply:
I'm not sure if we need to write down the medium term goal. In fact, I think we [already have](https://github.com/statechannels/statechannels/issues/3217)!